moveit_plugins/moveit_fake_controller_manager/src/moveit_fake_controller_manager.cpp

```diff
@@ -49,91 +47,126 @@ namespace moveit_fake_controller_manager
 static const std::string DEFAULT_TYPE = "interpolate";
 static const std::string ROBOT_DESCRIPTION = "robot_description";
 
+struct Controller_list {
```

Comment:
This changed a lot. We'll have to check if we can use all of these changes

Reply:
We should probably move this to the back and use a sim controller directly until then (i.e. using ros1_bridge)